misc/dockertest.py

```diff
@@ -271,6 +294,24 @@ def notary_list(fout, repo):
     return [line.strip().split() for line in lines[2:]]
 
 
+def test_build(fout, image, docker_version):
+    """
+    Build from a simple Dockerfile and ensure it works with DCT enabled
+    """
+    # build
+    # simple dockerfile to test building with trust
+    dockerfile = "FROM {0}:{1}\nRUN sh\n".format(image, docker_version)
+    tempdir_dockerfile = os.path.join(_TEMPDIR, "Dockerfile")
+    with open(tempdir_dockerfile, 'wb') as ftemp:
+      ftemp.write(dockerfile)
+
+    output = run_cmd_with_input(
+        "{0} build -".format(DOCKERS[docker_version]), tempdir_dockerfile, fout)
```

Comment:
Just curious - was just wondering why `cat <file> | docker build -` as opposed to `docker build -f <file>`?